### ISSUE TITLE
Thesis output adds a "10" to the first article in References

### ISSUE THREAD

**Issue by pachadotdev** [quarto-dev/quarto-cli#5926]

**Comment by cscheid** [comment]
Yes, this looks like a duplicate. The same comment from #5925 applies here. You're using a custom template and the .tex file generated by quarto doesn't have the spurious entry. I'm going to go ahead and close this one and we can continue the discussion in a single place.

**Comment by pachadotdev** [comment]
I got this. If I change where to include the `CSLReferences` definition, then Quarto won't create a tex that reads `\begin{{CSLReferences}{1}{0}` that is printed as a 10 in the references

solution here: https://github.com/quarto-dev/quarto-cli/issues/5926

**Comment by cscheid** [comment]
Ok, great! Happy you tracked it down!

The difference you're seeing between RMarkdown and Quarto, then, is likely coming from Pandoc, which uses `CSLReferences` (and you're probably using an older version of Pandoc in RMarkdown). Specifically, I think you're seeing this Pandoc bug: https://github.com/jgm/pandoc/issues/7531
